src/rubrix/client/datasets.py

```diff
@@ -535,6 +538,49 @@ def from_pandas(
     ) -> "DatasetForTokenClassification":
         return super().from_pandas(dataframe)
 
+    def prepare_for_training(self) -> "datasets.Dataset":
```

Comment:
I would use the `@_requires_datasets` decorator to get a helpful error message when `datasets` is not installed. 
```suggestion
    @_requires_datasets
    def prepare_for_training(self) -> "datasets.Dataset":
```